gui/src/Collaborators/components/projectMembers.tsx

```diff
@@ -25,6 +25,7 @@ import { connect } from 'react-redux';
 import '../styles/collaborators.scss';
 import { fetchMembersForProject } from '../actions/collaborators.actions';
 import { setCurrentProject } from '../../Projects/actions/projects.actions';
+import MouseEvent = __React.MouseEvent;
```

Comment:
Can this not be imported using `import { MouseEvent } from 'react'`? That syntax seems more idiomatic.